spotify_player/src/cli/handlers.rs

```diff
@@ -210,6 +210,7 @@ pub fn handle_cli_subcommand(cmd: &str, args: &ArgMatches) -> Result<()> {
         "like" => Request::Like {
             unlike: args.get_flag("unlike"),
         },
+        // TODO: handle search
```

Comment:
the pattern to handle a CLI command is 
1. defining a command like you already did in `spotify_player/src/cli/commands.rs`
2. defining a request variant of `Request`  in `spotify_player/src/cli/mod.rs`, for example `Search { query: string }`
3. construct a request based on CLI arguments, see how `Request::Like` is constructed above. For example, `Request::Search { query: args.get_one::<String>("query").expect("query is required").to_owned() }`
4. handle a request in `cli/client.rs`, you already defined a `handle_search_request` function

Reply:
you can take a look into how the app handles `get` command or other commands to understand the pattern

Reply:
Got it. Thanks for your suggestions and help on this :)